library/swift/src/RequestMapper.swift

```diff
@@ -0,0 +1,25 @@
+private let kRestrictedHeaderPrefix = ":"
+
+extension Request {
+  /// Returns a set of outbound headers that include HTTP
+  /// information on the URL, method, and additional headers.
+  ///
+  /// - returns: Outbound headers to send with an HTTP request.
+  func makeOutboundHeaders() -> [String: String] {
```

Comment:
`outboundHeaders()` to keep the method simple?

Reply:
Updated